src/ChunkMap.cpp

```diff
@@ -2593,6 +2593,34 @@ bool cChunkMap::ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinCh
 
 
 
+bool cChunkMap::ForEachLoadedChunk(cChunkDataCallback & a_Callback)
```

Comment:
What I was saying is that you have a variable (that implies there is some logic beind it) "Result" that is literally used for nothing apart from to store the value true.